tests/subsys/modbus/src/main.c

```diff
@@ -6,6 +6,14 @@
 
 #include "test_modbus.h"
 
+/*
+ * If need to test this test suite,
+ * should add the following fixtures to map.yml:
+ *
+ *   fixtures:
+ *   - uart_loopback
+ */
+
```

Comment:
There is already comment about test configuration in https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/subsys/modbus/src/test_modbus.h#L22, I would rather insert there: 
```diff
diff --git a/tests/subsys/modbus/src/test_modbus.h b/tests/subsys/modbus/src/test_modbus.h
index 0f38d7d908d..ce35d9dce3f 100644
--- a/tests/subsys/modbus/src/test_modbus.h
+++ b/tests/subsys/modbus/src/test_modbus.h
@@ -19,6 +19,9 @@
 #define MB_TEST_FP_OFFSET      5000
 
 /*
+ * This test performed on hardware requires two UART controllers
+ * on the board with RX/TX lines connected crosswise.
+ *
  * Integration platform for this test is FRDM-K64F.
  * The board must be prepared accordingly:
  * UART3(PTC16)-RX <-> UART2(PTD3)-TX pins and
```

Reply:
unresolved, :arrow_up: comment like this or NACK